stored_requests/config/config.go

```diff
@@ -101,45 +101,44 @@ func newFetchers(cfg *config.StoredRequests, client *http.Client, db *sql.DB) (f
 }
 
 func newCache(cfg *config.StoredRequests) stored_requests.Cache {
-	if cfg.InMemoryCache == nil {
+	if cfg.InMemoryCache.RequestCacheSize == 0 {
```

Comment:
eh... this will actually cause problems. See the docs on `InMemoryCache`:

```
// RequestCacheSize is the max number of bytes allowed in the cache for Stored Requests. Values <= 0 will have no limit
```

Reply:
I will have to add a `use_memory_cache` property, as it is a valid config to have all other cache properties be their zero values (just not all zero at the same time as that would be unlimited cache size with no expirery)

Reply:
yeah, that sounds right.

Alternatively you could add a `cfg.InMemoryCache.Type` with values `none`, `unbounded` and `lru`, and then `cfg.InMemoryCache.LRU` with the LRU-specific configs.

I just mention it because that pattern was used consistently in `prebid-cache`, and it worked out pretty well... but either one would work for now.